Tests/APITesters/CustomEntitlementComputationSwiftAPITester/SwiftAPITester/main.swift

```diff
@@ -42,6 +42,8 @@ func main() -> Int {
     checkPackageAPI()
     checkPackageEnums()
 
+    checkPresentedOfferingContextAPI()
```

Comment:
Looks like you haven't added these API changes to `CustomEntitlementComputationSwiftAPITester`.